src/main/java/org/fisco/bcos/channel/protocol/parser/BlkNotifyParser.java

```diff
@@ -0,0 +1,56 @@
+package org.fisco.bcos.channel.protocol.parser;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import java.io.IOException;
+import java.math.BigInteger;
+import org.fisco.bcos.channel.dto.BcosBlkNotify;
+import org.fisco.bcos.channel.protocol.Version;
+import org.fisco.bcos.web3j.protocol.ObjectMapperFactory;
+
+public class BlkNotifyParser {
```

Comment:
Do not use an abbreviation asap, BlockNotificationParser is more available

Reply:
> BlockNotificationParser

done